### DISCUSSION TITLE
Change Reply Settings on Bluesky Posts based on reading Article

### DISCUSSION THREAD

**Discussion by alexmcroberts** [joneslloyd/bluniversal-comments#6] [Ideas]
Original idea: https://bsky.app/profile/astrokatie.com/post/3lbvuaguins2b

Can reply settings of a post be set, or changed to people who read the article?

There's a few different directions this could take, but I'm not clear on what direction would be best!

**Comment by joneslloyd** [comment]
Thanks!

I'm thinking of how to make this happen
